### PR TITLE
Add options to send messages in different formatted style

### LINKED ISSUE #10: A discrepancy of message parsing between RC and EmbeddedChat
### Steps to reproduce:

1. Send `# h1`
2. It should be a heading
3. It shows as a heading in the EmbeddedChat but in RC -> `# h1`

### Expected behavior:
Markdown should work universally.

### Actual behavior:
It doesn't.

### LINKED ISSUE #98: Add options to send messages in different formatted style
As an EmbedChat developer

**I need to**: Different options available while sending a message. 
This will consist of following options:
1. Bold
2. Italic
3. Strike
4. Code
5. Multiline

**So That**: I can send in specified way as mention above

**Acceptance Criteria**

- [ ] Able to make message bold
- [ ] Able to italic text
- [ ] Able to strike text
- [ ] Able to include code
- [ ] Able to new line

### PR DESCRIPTION
**Pull request by imf-ali**
As an EmbedChat developer

**I need to**: Different options available while sending a message. 
This will consist of following options:
1. Bold
2. Italic
3. Strike
4. Code
5. Multiline

**So That**: I can send in specified way as mention above

**Acceptance Criteria**

- [x] Able to make message bold
- [x] Able to italic text
- [x] Able to strike text
- [x] Able to include code
- [x] Able to new line
- [x] Setup markdown used in RC core repo
- [x] Make message box height dynamic to certain height

Fix #98 
Fix #10 

[screen-capture.webm](https://user-images.githubusercontent.com/42451210/214566407-c175e7be-3366-4322-b602-746b983bec6b.webm)

### CODE REVIEW

File: src/components/ChatInput/ChatInput.js [hunk 1]
```diff
@@ -117,71 +105,60 @@ const ChatInput = ({ GOOGLE_CLIENT_ID }) => {
...
+      <Box className={styles.container}>
+        <textarea
+          placeholder={isUserAuthenticated ? 'Message' : 'Sign in to chat'}
           disabled={!isUserAuthenticated}
-          trigger={
+          className={styles.textInput}
+          onChange={(e) => {
+            messageRef.current.value = e.target.value;
+            setDisableButton(!messageRef.current.value.length);
+            e.target.style.height = 'auto';
```
Comment (sidmohanty11) [inline src/components/ChatInput/ChatInput.js:122]: can you please explain this part

File: package.json [hunk 2]
```diff
@@ -83,6 +83,7 @@
     "dompurify": "^2.3.8",
     "emoji-picker-react": "2.1.1",
     "emoji-toolkit": "^6.6.0",
+    "emojione": "^4.5.0",
```
Comment (sidmohanty11) [inline package.json:86]: can't we use `emoji-toolkit` for render? These are the newest versions for emoji-one I think

Refer: https://github.com/joypixels/emoji-toolkit/blob/master/examples/JAVASCRIPT.md

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
@imf-ali this definitely is amazing!

File: src/components/ChatInput/ChatInput.js (hunk 1 above)
Comment (imf-ali) [reply src/components/ChatInput/ChatInput.js:122]: In this part, whenever text contents increases then the box height, the box height also increases upto 140px after that it becomes constant. Its similar to how the height varies in RC server.

**Review by sidmohanty11 — APPROVED** [review]

**Pull request merged**